sofl/details_dialog.py

```
@@ -88,6 +89,16 @@ def __init__(self, game: Optional[Game] = None, **kwargs: Any):
             self.set_title(_("Add New Game"))
             self.apply_button.set_label(_("Add"))
 
+        if self.install_mode:
+            self.set_title(_("Install Game"))
```

Ruff (0.8.2)

Undefined name `_`

(F821)